week2/homework/src/update.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+function update(index, updatedText) {
+  fs.readFile('./src/data.json', 'utf8', (error, data) => {
```

Comment:
Is `.json` the right extension?

Reply:
in the assignment guadeline it is advised...

Reply:
That is true, but you are using plain text to store the data. Calling a file data.json does not magically make it a json file. In fact it is misleading as people will assume that the data in the file is stored in a json format and will try to parse it. If you are storing data as plain text the file should be data.txt